README.md

```diff
@@ -172,6 +172,7 @@ const client = new GraphQLClient(config)
 - `request(operation, options)`: Make a request to your GraphQL server; returning a Promise
   - `operation`: Object with `query`, `variables` and `operationName`
 - `options.fetchOptionsOverrides`: Object containing additional fetch options to be added to the default ones passed to `new GraphQLClient(config)`
+- `options.responseProxy`: Key-Value pairs to map parts of the original Fetch API response into the return value. Example for header forwarding would look like `{responseProxy: {header: resp => resp.headers}}`. Accessable is everything which the chosen Fetch implementation supports on the `Response` object.
```

Comment:
Hello!

What do you think of exposing a simple serialize function instead?

I'm thinking about something like this:
```js
{ responseReducer: (response) => ({ cookies: response.headers.get('cookies') }) }
```

This solution would be more performant, because we pick only what's needed.

Reply:
thanks for your feedbacks @gdorsi 🙏  I've updated the implementation and used a reducer instead.